src/style.js

```diff
@@ -465,7 +490,7 @@ const style = css`
   .speaker-select > paper-checkbox > span {
     font-weight: 600;
   }
-  paper-slider {
```

Comment:
Need to revert all these selectors back to paper-slider

Reply:
Please look again. All references have already been reverted. I think you looked at the wrong file.

Reply:
The HTML yes, but not the CSS selectors.

Reply:
Where do you see in the CSS ha-paper-slider? They are removed.

Reply:
ha-paper-slider? You changed some CSS selectors to ha-slider, you have to revert those back to paper-slider, I'll make comments on them for you.

Reply:
Yes. Sorry. Got confused with something else I was doing in HA main repo. Sorry.

Reply:
No worries, are there both a ha-paper-slider and a ha-slider in core?

Reply:
Yes unfortunately